front_end/content.py

```diff
@@ -911,8 +911,21 @@ def delete_problem_submissions(self, problem_basics):
 
         self.cursor.execute(sql, (problem_basics["assignment"]["course"]["id"], problem_basics["assignment"]["id"], problem_basics["id"],))
 
-    def export_course(self, course_basics, table_name, output_tsv_file_path):
-        sql = f"SELECT * FROM {table_name} WHERE course_id = ?"
+    def export_data(self, course_basics, table_name, output_tsv_file_path):
+        if table_name == "submissions":
```

Comment:
Please change these to inner joins. It probably won't make a difference, but it makes more sense conceptually for them to be inner joins.